app/config.py

```diff
@@ -507,13 +511,13 @@ class Preview(Config):
     NOTIFY_ENVIRONMENT = 'preview'
     CSV_UPLOAD_BUCKET_NAME = 'preview-notifications-csv-upload'
     CONTACT_LIST_BUCKET_NAME = 'preview-contact-list'
```

Comment:
Do the staging buckets need their own keys, too?

Reply:
I didn't think we were actually using the staging and preview environments? I keep meaning to clean up this file to remove unused environments.

Reply:
Yeah, we're using `live` for deployed code (hardcoded in manifest.yml). I created #67 to remove the unused environments.